script/jekyllmarkdowngen.py

```diff
@@ -476,11 +476,14 @@ def visit_type(self, typ: TypeDefinition) -> None:
                 full_path = sfx(self.namespaces._base) + (sfx(typ.imported_from) if typ.imported_from else '')
                 type_curie = self.namespaces.uri_or_curie_for(full_path, camelcase(typ.name))
                 type_uri = self.namespaces.uri_for(type_curie)
+
                 if type_curie.startswith('https://w3id.org/biolink/vocab/biolinkml:types/'):
                     ref = type_curie.split('/')[-1]
                     type_uri = f"https://biolink.github.io/biolinkml/docs/types/{ref}"
                     type_curie = f"metatype:{ref}"
-
+                if type_uri.startswith('https://w3id.org/biolink/vocab/'):
```

Comment:
I would recommend doing `elif` here since the previous `if` block will modify the CURIE and then the 2nd `if` block will modify it again. We don’t want that.

The first block is to handle primitive types in biolinkML, and so we want to be using the biolinkML W3ID URL.

The second block is to handle the types defined in biolink model and that’s where we want the `/vocab/types/` in the url.

Reply:
fixed. :)